projects/web/src/app/meetups-list/meetups-list.component.scss

```diff
@@ -0,0 +1,86 @@
+@import '../../theme';
+@import '../../mixins';
+@import '../../variables';
+
+main {
+  padding: 32px 22px 80px;
+
+  h1,
+  h2,
+  h3 {
+    color: mat-color($app-primary);
+  }
+
+  section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    a {
+      text-decoration: none;
+      width: 100%;
+      max-width: 400px;
+
+      .mat-card {
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-end;
+        margin: 8px;
+        cursor: pointer;
+
+        .mat-card-subtitle {
+          font-size: 14px;
+        }
+
+        .mat-card-image {
+          margin: 0 0 16px;
+          width: 100%;
+          height: 100%;
+          object-fit: cover;
+
+          &.svg {
+            width: 100%;
+            margin: auto;
+          }
+        }
+
+        .mat-card-actions {
+          padding: 0;
+
+          .mat-button {
+            text-transform: uppercase;
+          }
+        }
+      }
+    }
+  }
+
+  // Match the Angular Material SideNav's behavior
+  @include breakpoint('601px') {
```

Comment:
¿Qué tal si agregamos los tamaños de los viewports usando variables scss?

Se podría usar un archivo llamado breakpoints. Realmente entiendo que angular material tiene su gestor de layout pero podemos utilizar sus viewport por separado. Los break points están aquí:
https://github.com/angular/flex-layout/wiki/Responsive-API#mediaqueries-and-aliases

Que daría algo así.

Reply:
Estos tamaños ya están definidos en el archivo [_variables.scss](https://github.com/angular-hispano/angular-hispano/blob/master/projects/web/src/_variables.scss), por lo tanto esto si es algo para corregir ahora.

Reply:
Esto se debe solo a la coincidencia de los breakpoints utilizados por el Angular Material SideNav (como se menciona en el comentario). Esto no podemos usar uno de los breakpoints normales que ya hemos definido.